doc/developer-guide.md

```diff
@@ -31,7 +31,7 @@ Axe 3.0 supports open Shadow DOM: see our virtual DOM APIs and test utilities fo
 
 ### Environment Pre-requisites
 
-1. You must have NodeJS installed.
+1. You must have NodeJS version 12 or higher installed.
```

Comment:
Based on the current `.nvmrc`, this is not true; you must have 12.0.0 installed.